tests/conditional_compilation/conftest.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# pylint: disable=line-too-long
+
+""" Pytest configuration for compilation tests.
+
+Sample usage:
+python3 -m pytest --out ./compiled --models=<path to openvinotoolkit/testdata repository> \
+    --sea_runtool=./IntelSEAPI/runtool/sea_runtool.py \
+    --benchmark_app=./bin/benchmark_app test_collect.py
+"""
+
+import os
+import shutil
+import tempfile
+from types import SimpleNamespace
+
+import pytest
+
+# Using models from https://github.com/openvinotoolkit/testdata
+# $find models -wholename "*.xml"
+TESTS = [
+    {"path": "models/mobilenet_v2_1.4_224/mobilenet_v2_1.4_224_i8.xml"},
+    {"path": "models/mobilenet_v2_1.0_224/mobilenet_v2_1.0_224_i8.xml"},
+    {"path": "models/inception_v3/inception_v3_i8.xml"},
+    {"path": "models/resnet_v1_50/resnet_v1_50_i8.xml"},
+    {"path": "models/test_model/test_model_fp16.xml"},
+    {"path": "models/test_model/test_model_fp32.xml"},
+]
+
+
+def pytest_addoption(parser):
+    """ Define extra options for pytest options
+    """
+    parser.addoption("--models", default="", help="Path to models root directory")
+    parser.addoption(
+        "--sea_runtool", default="sea_runtool.py", help="Path to sea_runtool.py"
+    )
+    parser.addoption(
+        "--benchmark_app",
+        default="benchmark_app",
+        help="Path to the benchmark_app tool",
+    )
+    parser.addoption("--output", help="Output directory")
+
+
+def pytest_generate_tests(metafunc):
+    """ Generate tests depending on command line options
+    """
+    out = metafunc.config.getoption("output")
+    if out:
+        if os.path.isdir(out):
+            shutil.rmtree(out)  # cleanup output folder
+    else:
+        # generate unique output folder if `--output` option is not specified
+        out = tempfile.mkdtemp(prefix="_cc_out-", dir=".")
```

Comment:
this one should be moved to a separate fixture with `scope="function"` (or in test body)